server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/log.ftl

```diff
@@ -127,3 +127,6 @@
             <tbody></tbody>
         </table>
       </div>
+      <div>
+       <a href="javascript:clearLogs();javascript:refresh()">Clear Logs</a>
```

Comment:
Could this be turned into a bootstrap button with onclick action defined, instead of an HTML link to a javascript action? I think some browsers will have issues with the explicit href to javascript URIs.

Reply:
Working on this bootstrap button for you now Christopher.   The new code I have been adding has locked up the logs web page.